tests/data/dataconverter/readers/json_map/data.mapping.json

```diff
@@ -18,5 +18,6 @@
     "/ENTRY[entry]/optional_parent/required_child": "/required_child",
     "/ENTRY[entry]/program_name": "Example for listing exact data in the map file: Nexus Parser",
     "/ENTRY[entry]/required_group/description": "An example description",
-    "/ENTRY[entry]/required_group2/description": "An example description"
```

Comment:
I think it would also help to have another test without specifying the path `/ENTRY[entry]/optional_parent` to see whether it works to leave out the `optional_parent` group entirely.

Reply:
I added this test as well. It was quite needed. It works i guess.